content/cucumber/step-definitions.md

```diff
@@ -253,3 +253,48 @@ Given("I have {int} {color} balls", function (int1, color) {
 });
 ```
 {{% /block %}}
+
+# Abstract Your Steps
+
+It is always a good idea to make sure that you are not writing duplicate/similar step definitions. While there are a number of ways to do so and you can document your step definitions as well, it is always good to make use of **helper methods** to abstract your step definitions and reduce the number of steps.
```

Comment:
The content is good, but could be more concise. See if you can use less words to convey the same information. For example (but not limited to):
"It is always a good idea to make sure that you are not writing duplicate/similar step definitions." -> "Make sure that you are not writing very similar step definitions."

Reply:
Made it more concise. I ll keep checking and see if i can update it more, but take a look meanwhile, please :)